contrib/apelscripts/README.md

```diff
@@ -0,0 +1,15 @@
+HTCondor-CE APEL Scripts
+=========================
+
+This folder contains the HTCondor-CE scripts for extracting accounting log files suitable for input to the APEL system.
```

Comment:
We should explicitly mention that this works for an HTCondor + HTCondor-CE setup

Reply:
OK. WIlco.